contracts/test/integration/IntegrationTestMakerPCVSwapper.sol

```diff
@@ -0,0 +1,89 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {Vm} from "../unit/utils/Vm.sol";
+import {CoreV2} from "../../core/CoreV2.sol";
+import {DSTest} from "../unit/utils/DSTest.sol";
+import {MakerPCVSwapper} from "../../pcv/maker/MakerPCVSwapper.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {TestAddresses as addresses} from "../unit/utils/TestAddresses.sol";
+import {getCoreV2} from "./../unit/utils/Fixtures.sol";
+
+contract IntegrationTestMakerPCVSwapper is DSTest {
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    CoreV2 private core;
+    MakerPCVSwapper private swapper;
+    IERC20 private dai = IERC20(MainnetAddresses.DAI);
+    IERC20 private usdc = IERC20(MainnetAddresses.USDC);
+    uint256 private constant BALANCE = 1_000e18;
+
+    function setUp() public {
+        core = getCoreV2();
+        swapper = new MakerPCVSwapper(address(core));
+
+        // Get some DAI and USDC for tests
+        vm.startPrank(MainnetAddresses.DAI_USDC_USDT_CURVE_POOL);
+        dai.transfer(address(this), BALANCE);
+        usdc.transfer(address(this), BALANCE / 1e12);
+        vm.stopPrank();
```

Comment:
Once you're using Test instead of DSTest, would prefer if you could use deal instead of pranking as curve pool.